### ISSUE TITLE
InfluxDB error messages should be reported to the user

### ISSUE THREAD

**Issue by simonemainardi** [ntop/ntopng#2948]
INfluxDB can fail for example with:

```
CONTENT string {"error":"partial write: max-series-per-database limit exceeded: (1000000) dropped=41043"}
```

this error should be reported within the ntopng UI and possibly attached to the alert.

**Comment by simonemainardi** [comment]
similar error response:

```
RESPONSE_CODE number 400
CONTENT string {"error":"partial write: max-values-per-tag limit exceeded (100550/100000): measurement=\"host:traffic\" tag=\"host\" value=\"xxx\" dropped=40047"}
```

**Comment by simonemainardi** [comment]
other error:

```
 table
RESPONSE_CODE number 500
HTTP_HEADER table
CONTENT string {"error":"timeout"}

```

**Comment by emanuele-f** [comment]
Note: the message was previously extracted and used to generate an alert but later the alert was commented so such message is lost. Check out `_exportErrorMsg` which is still present in the code

**Comment by emanuele-f** [comment]
In 1fd65ac44c2ae84dfda8c977c079873f2d47fed3 the reporting of the message has been fixed. Alert generation must still be fixed though

**Comment by emanuele-f** [comment]
Alert fixed in 9833a87cbcfd82d1dc4c022e964559b397f4a986 . The error message is now checked every minute and an alert is generated.

![2020-01-28_15-08](https://user-images.githubusercontent.com/5488003/73270985-b2195480-41d7-11ea-8868-e65d4d645dc4.png)
